policies/coding-style.md

```diff
@@ -389,7 +409,11 @@ For example:
 
 ## Chapter 8: Commenting
 
-Use the classic "/* ... */" comment markers.  Don't use "// ..." markers.
+Use the classic `/* ... */` comment markers.  Don't use `// ...` markers.
+
+Place comments above or to the right of the code they refer to.
+Comments referring to the code line after
+should be indented equally to that code line.
```

Comment:
Do we need to clarify this more of where you expect to find the comment in case of an if / else? Do you write the comment before the if, or inside the if?

Reply:
I would say that it depends on the comment itself.  Does it explain the body, or does it explain why there's a condition?